receiver/swohostmetricsreceiver/internal/providers/infostat/provider.go

```diff
@@ -92,6 +92,6 @@ func (is *provider) provideInternal(ch chan<- InfoStat) {
 		HostID:               infoStat.HostID,
 	}
 
-	zap.L().Debug(fmt.Sprintf("InfoStat provided %+v", infoStatDetails))
+	is.logger.Debug("InfoStat provided", zap.Any("infoStatDetails", infoStatDetails))
```

Comment:
```suggestion
	is.logger.Debug("infostat provided", zap.Any("infostat_details", infoStatDetails))
```
Let's hold onto snake case in `zap.x()`.